elastic-search-support/src/main/resources/application.yml

```diff
@@ -33,6 +33,24 @@ elasticsearch:
                  }'
     alias: '{"type": "alias", "path": "data.<caseFieldPathPlaceHolder>"}'
     aliasTextSort: '{"type": "alias", "path": "data.<caseFieldPathPlaceHolder>.keyword"}'
+    ccdDynamicList: '{
+                        "properties": {
+                          "value": {
+                            "properties": {
+                              "code": {
+                                "type": "text" , "fields": { "keyword": { "type": "keyword", "ignore_above": 256}}
+                              },
+                              "label": {
+                                "type": "text" , "fields": { "keyword": { "type": "keyword", "ignore_above": 256}}
+                              }
+                            }
+                          },
+                          "list_items": {
+                            "type": "object",
+                            "enabled": false
+                          }
+                        }
+                      }'
```

Comment:
Indentation is a bit inconsistent (e.g. first and last don't line up).

Reply:
Updated the Indentation.